go.mod

```diff
@@ -377,13 +381,14 @@ replace (
 	github.com/pmezard/go-difflib => github.com/pmezard/go-difflib v1.0.0
 	github.com/pquerna/cachecontrol => github.com/pquerna/cachecontrol v0.0.0-20171018203845-0dec1b30a021
 	github.com/pquerna/ffjson => github.com/pquerna/ffjson v0.0.0-20180717144149-af8b230fcd20
-	github.com/prometheus/client_golang => github.com/prometheus/client_golang v0.9.4
+	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.0.0
```

Comment:
pending questions for @logicalhan or @brancz

Reply:
Oh I see.